news/2020-02-24-mappers-report-february.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: "The Mappers' Report: February"
+date: 2020-02-24 20:00:00 +0000
+---
+
+A new shiny news post was spotted at the horizon and coming to you now every month. It's **the Mappers' Report**! This news post series will inform you monthly about happenings within the osu! mapping scene.
+
+![](/wiki/shared/news/banners/the-mappers-report.jpg)
+
+Some of you may remember the old [QAT Gazette](/wiki/Quality_Assurance_Team_Blog) articles. This will be something similar, yet different. The Mappers' Report news posts will keep you up to date with Ranking Criteria changes, NAT meetings, and team movements. Additionally, we will include mapping related wiki changes.
+
+We even have something new in the works! The "Mapper Showcase" will be a section where each month we write about one mapper who deserves more attention. A mapper can be featured because of some unique mapping style, amazing storyboards or even just outstanding hitsounding. And don't worry, the game mode will switch every month. No one will be left out!
+
+With that being said, let's begin right away with the news!
+
+This month's issue of the Mappers' Report contains following topics:
+
+- **[Ranking Criteria Changes](#rcchanges)**
+- **[NAT Meetings](#natmeetings)**
+- **[Within the Beatmap Nominators and NAT](#withinbnnat)**
+- **[Mapping Contests](#mapping-contests)**
+- **[Summary](#summary)**
+
+## <a id="rcchanges"></a>Ranking Criteria Changes
+
+A new month brings a new avalanche of updates to the Ranking Criteria (RC). These past two months were especially productive with the implementation of rules!
+
+### General
```

Comment:
```suggestion
### All modes
```
more descriptive, "General" is ambiguous until you release that later sections are about modes